hmpps-prisoner-search/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/resource/PhysicalDetailResourceTest.kt

```diff
@@ -623,6 +623,16 @@ class PhysicalDetailResourceTest : AbstractSearchIntegrationTest() {
       expectedPrisoners = listOf("G7089EZ", "G7090AD"),
     )
 
+    // TODO I think this is wrong - neither prisoner has a single tattoo with both Knee and rose
+    @Test
+    fun `searching by tattoos with a body part and a comment on different tattoos`(): Unit = physicalDetailSearch(
+      detailRequest = PhysicalDetailRequest(
+        tattoos = listOf(BodyPart(bodyPart = "Knee", comment = "rose")),
+        prisonIds = listOf("AGI", "LEI"),
+      ),
+      expectedPrisoners = listOf("G7089EZ", "G7090AD"),
+    )
```

Comment:
I was wondering why Physical Details Search didn't have the same problem - but it does. So I added a test to highlight the issue that will need updating when we make tattoos a nested object.